DigitalLearningSolutions.Web/Scripts/trackingSystem/emailDelegates.ts

```diff
@@ -8,6 +8,12 @@ const route = `TrackingSystem/Delegates/Email/AllEmailDelegateItems?${queryParam
 // eslint-disable-next-line no-new
 new SearchSortFilterAndPaginate(route, false, false, true, 'EmailDelegateFilter');
 
+function alertResultCount(): void {
+  const resultCount = document.getElementById('results-count') as HTMLSpanElement;
+  resultCount.setAttribute('role', '');
+  resultCount.setAttribute('role', 'alert');

```

Comment:
What does this do? Maybe worth a comment if this is a hack to let screen reader announce the message/or rename to announceNewResultCount()?
Alert usually means popup on a page in terms of JS (try the alert method in console):
![image](https://user-images.githubusercontent.com/40016796/134702907-e7a807f0-86c4-4495-9957-82f18835099d.png)

Reply:
If a html element has role=alert it gets read by the screen reader. In the filtering stuff, whenever a filter is changed, the text on the element changes which makes it re-read. Here the text isn't changing, so we can just unassign and reassign the role alert to make it re-read.